src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionOperatorsTests.cs

```diff
@@ -567,23 +605,238 @@ static class Extensions3
     }
 }
 
+static class Extensions4
+{
+    extension(ref S1? s1)
+    {
+        public void operator {{{op}}}() {}
```

Comment:
nit: consider adding number comments (`// N`) to make it easier to review this long test

Reply:
> consider adding number comments 

I am using ```#line``` directives, not planning to use comments because they are creating a maintenance issues since some devs confuse them with form of a base-line